apps/client/src/widgets/type_widgets/editable_text.ts

```diff
@@ -287,7 +287,7 @@ export default class EditableTextTypeWidget extends AbstractTextTypeWidget {
 
             // Touch bar integration
             if (hasTouchBar) {
-                for (const event of [ "bold", "italic", "underline", "paragraph", "heading" ]) {
+                for (const event of ["bold", "italic", "underline", "paragraph", "heading"]) {
```

Comment:
coding style shouldn't have been changed here, that spacing around these brackets was "on purpose" due to the projects coding style rules